navigation.py

```diff
@@ -42,12 +42,40 @@ def navigate_to_link(link, driver):
     # wait for page to load, up to ten seconds
     WebDriverWait(driver, 10).until(EC.presence_of_element_located((By.XPATH, "//html")))
 
-def assert_current_page_is(page_name, driver):
-    if page_name in pages:
-        assert driver.current_url == pages[page_name]
-    else:
+def assert_current_page_is(page_name, driver, retry=0):
+    """ Assert that current url is as given.
```

Comment:
"Assert that the current url is the one for "page_name"" would be more descriptive.